src/Repository/Competition/CompetitionRepository.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace BalticRobo\Api\Repository\Competition;
+
+use BalticRobo\Api\Entity\Competition\Competition;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @method Competition|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Competition|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Competition[]    findAll()
+ * @method Competition[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
```

Comment:
Remove that, we won't use `find()` methods. You should call method which is descriptive.